README.md

```diff
@@ -20,98 +17,20 @@ To run this CLI application
-- [Git](https://git-scm.com/downloads)
-
-## Quickstart
-
-To get started with Scaffold-ETH 2, follow the steps below:
-
-1. Install from NPM Registry and follow the CLI instructions.
-
-```
-npx create-eth@latest
-```
-
-> 💬 Hint: If you choose Foundry as solidity framework in the CLI, you'll also need Foundryup installed in your machine. Checkout: [getfoundry.sh](https://getfoundry.sh)
-
-2. Run a local network in the first terminal:
-
-```
-yarn chain
-```
-
-This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:
-
-- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
-- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
-
-This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
-
-- Hardhat => `packages/hardhat/contracts`
-- Foundry => `packages/foundry/contracts`
-
-The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:
-
-- Hardhat => `packages/hardhat/deploy`
-- Foundry => `packages/foundry/script`
-
-4. On a third terminal, start your NextJS app:
-
-```
-yarn start
-```
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
```

Comment:
I'm having trouble picturing what this would look like in the terminal. Would it be a list of every challenge name and they can up/down arrow to select?

Reply:
Hmm this is interesting. Firstly the high-level comments/questions & then detailed stuff.

## High-Level
1. All three of us discussed the concept of the terminal vs a front end. Where did we land on going with the CLI vs the front-end again? Is it really so interesting that we deem it necessary vs a front end that has this graphic that you are describing with interactive links that stitch with certain checks for a user's experience, etc.?
   - I am not dunking on the CLI, I am just playing devil's advocate.
2. How are we tracking a user's experience? Is it via NFTs? Is it on-chain somehow? If we make it on-chain based, then I'm wondering if we can stitch it with the BUIDL GUIDL builder profiles somehow. When looking at the BUIDL GUIDL we can see that builders have milestones (completed the DEX challenge), etc. Can we use this same system to be consistent, and to possibly make figuring out this experience tracker side of things possibly easier (since it's been done like this already)?
3. Students often take paths that aren't prescribed. So giving the option for students to pave their own way is interesting (taking on different challenges, and earning experience vs having to go through levels). Perhaps we can do both, where XP is tracked and different options are available, but they aren't necessarily allowed to do some insanely hard challenge that isn't related. UNLESS they do a personal challenge or something to prove themself.

## Details
1. It would be cool if the BUIDL GUIDL profiles become more of a resume for students. AKA can we stitch the UX so a student has to upload their build on their profiles (or it auto-uploads) upon submitting their challenge?
2. To be sure we're on the same page, maybe @escottalexander can draw up a quick sketch on an iPad or something of what he envisions with the terminal?

Reply:
If we are going with the CLI, it would be great to just get a sketch of what this will look like on the README (using figma or just an iPad idk). If it's faster to just make with the tech tree, that's chill.